lib/l10n/app_en.arb

```diff
@@ -7,6 +7,7 @@
   "extremely_likely": "Extremely Likely",
   "start_survey": "Start Survey",
   "submit_survey": "Submit Survey",
+  "thank_you_survey": "Thanks for taking the survey.",
```

Comment:
I think rather than using static text we can use outro text from this [PR](https://github.com/nimblehq/ic-flutter-taher-toby/pull/90) , as the outro text is different for each survey. What do you think?

Reply:
I'll keep this is the default value in case the `outro text` is null, what do you think? 🙏 

Will refactor that in the next PR 🚀